### PR TITLE
Uifc 153 add assigned unassigned filter

### PR DESCRIPTION
**Pull request by elsenhans**
Richard, it seems to be working. What do you think?

### CODE REVIEW

File: src/CollectionSearchContainer.js [hunk 1]
```diff
@@ -103,25 +136,41 @@ class CollectionSearchContainer extends React.Component {
...
     const { onSelectRow, resources } = this.props;
-
+    const contentData = _.get(resources, 'metadataCollections.records', []);
+    const filterToCollections = _.get(resources, 'filterToCollections.records', []);
     if (this.collection) {
       this.collection.update(this.props, 'metadataCollections');
     }
 
+    // Here we filter collections is they are assigned or unassigned
+    // MAybe this block is better suited in CollectionsSearchContainer.js?
```
Comment (rchr) [inline src/CollectionSearchContainer.js:154]: Can you please remove the comments, that may be ambiguous in the future as well as the commented log statements?

File: src/CollectionSearchContainer.js [hunk 2]
```diff
@@ -103,25 +136,41 @@ class CollectionSearchContainer extends React.Component {
...
+    // console.log(`Here you can filter your contenData by filterCollections ${filterToCollections} and the assignedStatus ${assignedStatus}`);
+    let filtered = contentData;
+    if (_.findIndex(this.state.assignedStatus, s => s.includes('yes')) >= 0 && _.findIndex(this.state.assignedStatus, s => s.includes('no')) === -1) {
+      filtered = contentData.filter(c => filterToCollections.includes(c.id));
+      // console.log(`The assigned collections are ${filtered.map(c => c.id).join(', ')}`);
+    } else if (_.findIndex(this.state.assignedStatus, s => s.includes('no')) >= 0 && _.findIndex(this.state.assignedStatus, s => s.includes('yes')) === -1) {
+      filtered = contentData.filter(c => !filterToCollections.includes(c.id));
+      // console.log(`The unassigned collections are ${filtered.map(c => c.id).join(', ')}`);
+    }
+    // I am a bit unsure if we can safely replace count by this statement
```
Comment (rchr) [inline src/CollectionSearchContainer.js:164]: This line and the following may be ambiguous as well...

File: src/CollectionSearchContainer.js (hunk 1 above)
Comment (elsenhans) [reply src/CollectionSearchContainer.js]: sure, but the first line with "// Here we filter collections is they are assigned or unassigned" may be quite useful. Could it stay?

Comment (rchr) [reply src/CollectionSearchContainer.js]: Yes, the 

> // Here we filter collections is they are assigned or unassigned" 

is quite useful. Please keep it!

**Pull request merged**